sdks/python/apache_beam/transforms/xlang/__init__.py

```diff
@@ -16,6 +16,12 @@
 #
 
 """
-This is a subdirectory for generated external transforms.
-Refer to sdks/python/gen_xlang_wrappers.py for more details.
+This package contains autogenerated Python wrappers for cross-language
+transforms available in other Beam SDKs.
+
+For documentation on creating and using cross-language transforms, see:
+https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines
```

Comment:
1) would it make sense to  reference a link to pydoc here? 
2) https://beam.apache.org/documentation/programming-guide/#use-x-lang-transformsShould were refernce the 'using' portion specifically?  https://beam.apache.org/documentation/programming-guide/#use-x-lang-transforms

Reply:
1. The script won't show up on the pydoc, so I'm not sure what we can reference here? 
P.S. the suggestion in this [comment](https://github.com/apache/beam/pull/29834/#discussion_r1476494188) would be adding a self-referencing link
2. SGTM